### ISSUE TITLE
Edit AFK Mob Grind sword interval.

### ISSUE THREAD

**Issue by sasspaper** [monpjc/XAHK#43]
The current sweep speed for the sword in the AFK Mob option is very very slow, and I couldn't find a way to change it. Is there plans to change this?

**Comment by histefanhere** [comment]
Yes there is a way to change it, but not in the compiled executable. You would have to install AutoHotkey from [their website](https://www.autohotkey.com/) (The script still uses v1 NOT v2 of AHK (for now)), download the repository and edit the script, then run the script file or compile it to a new executable. You want to change the `Delay >= 12` in line 384 of the code as shown here:

https://github.com/histefanhere/XAHK/blob/cf445ece2d3a05653d39c860cb7cb6bb15d9635d/minescript.ahk#L384

And yes there are plans to make *every* parameter adjustable straight from the GUI, but that will be who knows when - hopefully sooner than later 😔

**Comment by sasspaper** [comment]
Thanks!
